Unable to install on Windows 10

```
npm install @shopify/react-native-skia
```
gives me this error

```
npm WARN ERESOLVE overriding peer dependency
npm ERR! code 1
npm ERR! path C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node scripts/install-npm.js
npm ERR! Updating symlinks for Android build...
npm ERR! Creating symlink to api C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia\scripts C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia
npm ERR! internal/fs/utils.js:332
npm ERR!     throw err;
npm ERR!     ^
npm ERR!
npm ERR! Error: EPERM: operation not permitted, symlink 'C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia\cpp\api' -> 'C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia\android\cpp\api'
npm ERR!     at Object.symlinkSync (fs.js:1214:3)
npm ERR!     at createSymlink (C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia\scripts\install-npm.js:12:8)
npm ERR!     at Object.<anonymous> (C:\dev\medion\base-app\apps\robots\node_modules\@shopify\react-native-skia\scripts\install-npm.js:17:1)
npm ERR!     at Module._compile (internal/modules/cjs/loader.js:1085:14)
npm ERR!     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
npm ERR!     at Module.load (internal/modules/cjs/loader.js:950:32)
npm ERR!     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
npm ERR!     at internal/main/run_main_module.js:17:47 {
npm ERR!   errno: -4048,
npm ERR!   syscall: 'symlink',
npm ERR!   code: 'EPERM',
npm ERR!   path: 'C:\\dev\\medion\\base-app\\apps\\robots\\node_modules\\@shopify\\react-native-skia\\cpp\\api',
npm ERR!   dest: 'C:\\dev\\medion\\base-app\\apps\\robots\\node_modules\\@shopify\\react-native-skia\\android\\cpp\\api'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sauer\AppData\Local\npm-cache\_logs\2022-04-19T06_59_41_888Z-debug-0.log
```

I guess its due to this issue https://github.com/nodejs/node/issues/22208 maybe?

--- a/package/scripts/install-npm.js
+++ b/package/scripts/install-npm.js
@@ -9,7 +9,7 @@ const createSymlink = (p) => {
   const dstDir = path.resolve(`./android/cpp/${p}`);
 
   if (!fs.existsSync(dstDir) || !fs.lstatSync(dstDir).isSymbolicLink()) {
-    fs.symlinkSync(srcDir, dstDir, "dir");
+    fs.symlinkSync(srcDir, dstDir, "junction");
   }
 };
 

